resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/submit/ClientSuite.scala

```diff
@@ -337,4 +337,32 @@ class ClientSuite extends SparkFunSuite with BeforeAndAfter {
     submissionClient.run()
     verify(loggingPodStatusWatcher).watchOrStop(kconf.namespace + ":driver")
   }
+
+  test("driver's confimap.metadata.namespace set to conf.namespace otherwise default") {
```

Comment:
nit: test("SPARK-37916: driver's confimap.metadata.namespace set to conf.namespace otherwise default") {